<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inicio</class>
 <widget class="QWidget" name="Inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>459</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img_fundoTelaBemVindo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>850</width>
     <height>450</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>899</width>
     <height>500</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons/img_inicio.qrc">:/img_inicio/img_telaLogin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_criarConta">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_criarConta">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 600 9pt &quot;Segoe UI&quot;;
    color: rgb(255, 255, 255);
    background-color: rgb(197, 68, 76);
    border-radius: 10px;
}

QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: rgb(197, 68, 76);
}</string>
   </property>
   <property name="text">
    <string>Criar conta</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_chamada">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 900 16pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Confira os filmes em cartaz nesta semana</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

font: 600 9pt &quot;Segoe UI&quot;;
 color: rgb(197, 68, 76);
 background-color: rgb(240, 240, 255);
 border-radius: 10px;
}

QPushButton:hover {
    background-color: rgb(197, 68, 76);
    color: rgb(255, 255, 255);

}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_logo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;Bem Vindo&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#c5444c;&quot;&gt; PobreVision&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons/img_inicio.qrc"/>
 </resources>
 <connections/>
</ui>
